lib/main_common.pm

```diff
@@ -2081,12 +2084,23 @@ sub load_publiccloud_tests {
     loadtest 'publiccloud/run_ltp'       if get_var('PUBLIC_CLOUD_LTP');
 }
 
+# Scheduling set for validation of specific installation
+sub load_installation_validation_tests {
+    load_system_prepare_tests;
+    load_system_prepare_tests;
+    for my $module (split(',', get_var('INSTALLATION_VALIDATION'))) {
+        # Has to introduce this mapping because Travis checks if test module is used anywhere
```

Comment:
That should not be necessary. Just adjust the tests which are BTW not specific to Travis but you can call them independently

Reply:
Hmm, I didn't get this part. We have a module which is never mentioned in the `main.pm`, so how can I adjust the test, so travis doesn't fail?